roles/mongodb/tasks/mongodb_rhel6_dependencies.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Install mongodb dependencies for rhel6
+  become: yes
+  yum: name={{item}} state=present
+  with_items:
+## Failed to validate the SSL certificate for www.mongodb.org:443. Make sure your managed systems have a valid CA certificate installed. If the website serving the url uses SNI you need python >= 2.7.9 on your managed machine or you can install the `urllib3`, `pyopenssl`, `ndg-httpsclient`, and `pyasn1` python modules to perform SNI verification in python >= 2.6. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible
```

Comment:
Yeeah, looks familiar, got it for deb once:
https://github.com/StackStorm/ansible-st2/commit/b951004e4e6425626de596c4d47454a5a2d2dd43
Just FYI